Lib/_pyio.py

```diff
@@ -766,7 +766,7 @@ def truncate(self, pos=None):
 
     def flush(self):
         if self.closed:
-            raise ValueError("flush of closed file")
+            raise ValueError("flush on closed file")
```

Comment:
This is not a typo, it is a wording choice.  I don't see the point in changing it, and in fact as a native English speaker I think 'of' reads better here.  To you 'on' you'd want to say "flush called on closed file object", which is wordier.